docs/content/preview/manage/backup-restore/instant-db-cloning.md

```diff
@@ -0,0 +1,292 @@
+    production_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+    k | v
+   ---+---
+    1 | 1
+    2 | 0
+    3 | 1
+    4 | 0
+    5 | 1
+   (5 rows)
+    ```
+
+    ```sql
+    production_db=# CREATE TABLE t2 (key INT, c1 TEXT);
+    production_db=# INSERT INTO t2 (key,c1) SELECT i,md5(random()::text) FROM generate_series(1,5) AS i;
+    production_db=# SELECT * FROM t2 ORDER BY key;
+    ```
+
+    ```output
+     key |                c1
+    -----+----------------------------------
+       1 | 450e6c49f86c76d944375e29e48f2dee
+       2 | b934a3bdf7438458a85b0858c41f731c
+       3 | 08697ed89ec387e714c6587e522d7a7e
+       4 | a879ff99872b3c3433803d3c3229f0cf
+       5 | 4d46a53780a7a348179e1af9b692e95e
+    (5 rows)
+    ```
+
+1. Determine the exact time when your database is in the correct state. This timestamp will be used to create a clone of the production database from the point when it was in the desired state. Execute the following SQL query to retrieve the current time in UNIX timestamp format:
+
+    ```sql
+    production_db=# SELECT (EXTRACT (EPOCH FROM CURRENT_TIMESTAMP)*1000000)::decimal(38,0);
+    ```
+
+    ```output
+         numeric
+     ------------------
+      1723243720285350
+      (1 row)
+    ```
+
+1. Drop the table `t1` simulating a user error dropping the table accidentally.
+
+    ```sql
+    production_db=# DROP TABLE t1;
+    ```
+
+    ```output
+    DROP TABLE
+    ```
+
+1. Meanwhile, as table `t2` is still accepting reads/writes, you can insert 2 more rows as follows:
+
+    ```sh
+    INSERT INTO t2 (key,c1) SELECT i,md5(random()::text) FROM generate_series(6,7) AS i;
+    ```
+
+1. Now, if you try to query table `t1`, notice that the table is dropped and there is no way you can query it.
+
+    ```sql
+    production_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+    ERROR:  relation "t1" does not exist
+    LINE 1: SELECT * FROM t1 ORDER BY k;
+    ```
+
+1. To get back lost data, create a database `clone_db` using `production_db` as the template and using the timestamp generated in step 4.
+
+    ```sql
+    production_db=# CREATE DATABASE clone_db TEMPLATE production_DB AS OF 1723243720285350;
+    ```
+
+    ```sql
+    \c clone_db
+    ```
+
+    ```output
+    You are now connected to database "clone_db" as user "yugabyte".
+    ```
+
+    ```sql
+    clone_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+     k | v
+    ---+---
+     1 | 1
+     2 | 0
+     3 | 1
+     4 | 0
+     5 | 1
+    (5 rows)
+    ```
+
+    `clone_db` contains all the data from `production_db` at the specified timestamp, which means you can read table `t1` that was dropped accidentally. You can copy the lost data from table `t1` back to `production_db` by exporting the data from the clone and importing it into `production_db`. Alternatively, you can switch the workload to `clone_db`. You now have two isolated databases that can serve reads and writes independently.
+
```

Comment:
When you are done, you can clean up by dropping the clone database as you would any database, using the DROP DATABASE or DROP KEYSPACE command. The clone is deleted, along with any post-compaction uncompacted files from the original database.